action/test/main.spec.ts

```diff
@@ -25,11 +25,68 @@ describe('Frogbot Action Tests', () => {
         test.each(cases)('CLI Url for %s-%s', (platform, arch, fileName, expectedUrl) => {
             myOs.platform.mockImplementation(() => <NodeJS.Platform>platform);
             myOs.arch.mockImplementation(() => arch);
-            let cliUrl: string = Utils.getCliUrl('1', '1.2.3', fileName);
+            let cliUrl: string = Utils.getCliUrl('1', '1.2.3', fileName, '');
             expect(cliUrl).toBe(expectedUrl);
         });
     });
 
+    describe('Frogbot URL Tests With Remote Artifactory', () => {
+        const myOs: jest.Mocked<typeof os> = os as any;
+        const releasesRepo: string = 'frogbot-remote';
+        process.env['JF_URL'] = 'https://myfrogbot.com/';
+        process.env['JF_ACCESS_TOKEN'] = 'access_token1';
+        let cases: string[][] = [
+            [
+                'win32' as NodeJS.Platform,
+                'amd64',
+                'jfrog.exe',
+                'https://myfrogbot.com/artifactory/frogbot-remote/artifactory/frogbot/v2/2.8.7/frogbot-windows-amd64/jfrog.exe',
+            ],
+            ['darwin' as NodeJS.Platform, 'amd64', 'jfrog', 'https://myfrogbot.com/artifactory/frogbot-remote/artifactory/frogbot/v2/2.8.7/frogbot-mac-386/jfrog'],
+            ['linux' as NodeJS.Platform, 'amd64', 'jfrog', 'https://myfrogbot.com/artifactory/frogbot-remote/artifactory/frogbot/v2/2.8.7/frogbot-linux-amd64/jfrog'],
+            ['linux' as NodeJS.Platform, 'arm64', 'jfrog', 'https://myfrogbot.com/artifactory/frogbot-remote/artifactory/frogbot/v2/2.8.7/frogbot-linux-arm64/jfrog'],
+            ['linux' as NodeJS.Platform, '386', 'jfrog', 'https://myfrogbot.com/artifactory/frogbot-remote/artifactory/frogbot/v2/2.8.7/frogbot-linux-386/jfrog'],
+            ['linux' as NodeJS.Platform, 'arm', 'jfrog', 'https://myfrogbot.com/artifactory/frogbot-remote/artifactory/frogbot/v2/2.8.7/frogbot-linux-arm/jfrog'],
+            ['linux' as NodeJS.Platform, 'ppc64', 'jfrog', 'https://myfrogbot.com/artifactory/frogbot-remote/artifactory/frogbot/v2/2.8.7/frogbot-linux-ppc64/jfrog'],
+            ['linux' as NodeJS.Platform, 'ppc64le', 'jfrog', 'https://myfrogbot.com/artifactory/frogbot-remote/artifactory/frogbot/v2/2.8.7/frogbot-linux-ppc64le/jfrog'],
+        ];
+
+        test.each(cases)('Remote CLI Url for %s-%s', (platform, arch, fileName, expectedUrl) => {
+            myOs.platform.mockImplementation(() => <NodeJS.Platform>platform);
+            myOs.arch.mockImplementation(() => arch);
+            let cliUrl: string = Utils.getCliUrl('2', '2.8.7', fileName, releasesRepo);
+            expect(cliUrl).toBe(expectedUrl);
+        });
+    });
+
+    describe('generateAuthString', () => {
+        it('should return an empty string if releasesRepo is falsy', () => {
```

Comment:
Let's be consistent with the names of the other tests by capitalizing the test names.
For example:
```ts
describe('Generate auth string', () => {
```

```ts
it('Should return an empty string if releasesRepo is falsy', () => {
```

Reply:
Done